zheng-common/src/test/java/com/zheng/common/util/PaginatorTest.java

```diff
@@ -0,0 +1,116 @@
+package com.zheng.common.util;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.lang.reflect.Method;
+import javax.servlet.http.HttpServletRequest;
+
+@RunWith(PowerMockRunner.class)
+public class PaginatorTest {
+
+  @PrepareForTest({Paginator.class, HttpServletRequest.class})
+  @Test
+  public void constructor() throws Exception {
```

Comment:
We don't usually submi tests for constructors

Reply:
This one has interesting side-effects that to assert, though.

Reply:
@allredj Run it past @peterschrammel

Reply:
I think this is interesting enough to submit.